internal/storage/remote_chunk_manager.go

```diff
@@ -45,11 +45,25 @@ const (
 	CloudProviderAzure = "azure"
 )
 
+const NoSuchKey = "NoSuchKey"
+
+var ErrNoSuchKey = errors.New(NoSuchKey)
+
+func WrapErrNoSuchKey(key string) error {
+	return fmt.Errorf("%w(key=%s)", ErrNoSuchKey, key)
+}
+
+func IsErrNoSuchKey(err error) bool {
```

Comment:
seems unused, along with `WrapErrNoSuchKey`, `ErrNoSuchKey`, and `NoSuchKey`.

Reply:
It's removed at master, but still using on 2.3 branch.
because I remove the redundant `internal/storage/minio_chunk_manager.go` file, move the original code here.